clients/lsp-zig.el

```diff
@@ -26,9 +26,21 @@
 
 (require 'lsp-mode)
 
+(defgroup lsp-zig nil
+  "LSP support for Zig via zls."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/zigtools/zls"))
+
+(defcustom lsp-clients-zls-executable "zls"
```

Comment:
This should be named something like with `lsp-zig` prefix
```suggestion
(defcustom lsp-zig-zls-executable "zls"
```

Reply:
I was actually following some of the other LSP clients in following the `lsp-clients-` prefix naming scheme. I can also update this in a PR if desired.

Reply:
Yeah, the `lsp-clients-` prefix is from when we have a gigantic `lsp-clients.el` file and group.
Now it's separated into several smaller one, this should be prefixed as group name